Example/GiniVision/AnalysisManager.swift

```diff
@@ -135,51 +135,51 @@ class AnalysisManager {
             return self.document?.extractions
             
         // 4. Handle results
-        }).continueWithBlock({ (task: BFTask!) -> AnyObject! in
-            if token.cancelled || task.cancelled {
+        }).continue({ (task: BFTask?) -> AnyObject! in
+            if token.cancelled || (task?.isCancelled == true) {
                 print("Canceled analysis process")
-                return BFTask.cancelledTask()
+                return BFTask.cancelled()
             }
             
             print("Finished analysis process")
             
-            let userInfo: [NSObject: AnyObject]
+            let userInfo: [AnyHashable: Any]
             let notificationName: String
             
-            if let error = task.error {
+            if let error = task?.error {
                 self.error = error
                 userInfo = [ GINIAnalysisManagerErrorUserInfoKey: error ]
                 notificationName = GINIAnalysisManagerDidReceiveErrorNotification
-                completion?(inner: { _ in throw error })
+                completion?({ _ in throw error })
             } else if let document = self.document,
-                      let result = task.result as? GINIResult {
+                      let result = task?.result as? GINIResult {
                 self.result = result
                 self.document = document
                 userInfo = [
                     GINIAnalysisManagerResultDictionaryUserInfoKey: result,
                     GINIAnalysisManagerDocumentUserInfoKey: document
                 ]
                 notificationName = GINIAnalysisManagerDidReceiveResultNotification
-                completion?(inner: { _ in return (result, document) })
+                completion?({ _ in return (result, document) })
             } else {
-                enum AnalysisError: ErrorType {
-                    case Unknown
+                enum AnalysisError: Error {
+                    case unknown
                 }
-                let error = NSError(domain: "net.gini.error.", code: AnalysisError.Unknown._code, userInfo: nil)
+                let error = NSError(domain: "net.gini.error.", code: AnalysisError.unknown._code, userInfo: nil)
                 self.error = error
                 userInfo = [ GINIAnalysisManagerErrorUserInfoKey: error ]
                 notificationName = GINIAnalysisManagerDidReceiveErrorNotification
-                completion?(inner: { _ in throw AnalysisError.Unknown })
+                completion?({ _ in throw AnalysisError.unknown })
                 return nil
             }
-            dispatch_async(dispatch_get_main_queue(), {
-                NSNotificationCenter.defaultCenter().postNotificationName(notificationName, object: self, userInfo: userInfo)
+            DispatchQueue.main.async(execute: {
```

Comment:
Let's be consistent throughout the app and framework and omit the braces `(execute: ... )`.